src/execution_plan/execution_plan_modify.c

```diff
@@ -313,11 +315,41 @@ void ExecutionPlan_BindPlanToOps(ExecutionPlan *plan, OpBase *root) {
 	}
 }
 
-void ExecutionPlan_AppendSubExecutionPlan(ExecutionPlan *master_plan, ExecutionPlan *sub_plan) {
-	if(!master_plan->sub_execution_plans)
-		master_plan->sub_execution_plans = array_new(ExecutionPlan *, 1);
-	if(sub_plan->record_map) raxFree(sub_plan->record_map);
-	sub_plan->record_map = master_plan->record_map;
-	master_plan->sub_execution_plans = array_append(master_plan->sub_execution_plans, sub_plan);
+OpBase *ExecutionPlan_BuildOpsFromPath(ExecutionPlan *plan, const char **vars,
```

Comment:
rename `vars` to `bound_vars` or anything else which makes this a bit more clear.